distest-test-bot.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3.7
+
+# Based on the example tester at
+# https://github.com/JakeCover/distest/blob/develop/example_tester.py
+#
+# To run:
+# ./distest-test-bot.py --run all --channel <channel ID> 821891815329890336 <tester token>
```

Comment:
I like seeing the run command in the comment. Please consider adding this to the readme file.

Reply:
I plan to. See Action #78 (just created today) which I'll take care of around the end of next week as things are wrapping up.